lunr/utils.py

```diff
@@ -1,7 +1,13 @@
-def as_string(obj):
-    return "" if not obj else str(obj)
+from typing import Any
 
 
+def as_string(obj: Any):
+    return "" if obj is None else str(obj)
```

Comment:
This change is because, before, if `obj` was 0, it would return "" instead of "0", which could cause some recall failures in search.